docs/resources/synciq_replication_job.md

```diff
@@ -19,22 +19,59 @@ linkTitle: "powerscale_synciq_replication_job"
 page_title: "powerscale_synciq_replication_job Resource - terraform-provider-powerscale"
 subcategory: ""
 description: |-
-  Resource for managing SyncIQReplicationJobResource on OpenManage Enterprise.
+  Resource for managing SyncIQ ReplicationJob on PowerScale. This resource can be used to manually trigger the replication job to replicate data from source powerscale cluster to a target powerscale cluster.
 ---
 
 # powerscale_synciq_replication_job (Resource)
 
-Resource for managing SyncIQReplicationJobResource on OpenManage Enterprise.
+Resource for managing SyncIQ ReplicationJob on PowerScale. This resource can be used to manually trigger the replication job to replicate data from source powerscale cluster to a target powerscale cluster.
 
 
+## Example Usage
 
+```terraform
+/*
+Copyright (c) 2024 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# Available actions: Create and  Update updates the syncIQ Replication Job. Delete will delete job and clear the state file. 
+# After `terraform apply` of this example file will perform action on the synciq replicaiton job according to the attributes set in the config
+
+# PowerScale SynIQ Replication Job allows you to manage the SyncIQ Replication Jobs on the Powerscale array
+resource "powerscale_synciq_replication_job" "job1" {
```

Comment:
we should give more examples explain the work of each action.. It will make things clear

Reply:
Added the guide